website/screens/components/header/usage/HeaderUsagePage.tsx

```diff
@@ -4,8 +4,9 @@ import Figure from "@/common/Figure";
 import Image from "@/common/Image";
 import QuickNavContainer from "@/common/QuickNavContainer";
 import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
-import headerVariants from "./images/header_variants.png";
 import headerResponsive from "./images/header_responsive.png";
```

Comment:
Please remove also the `header_variants` image from `/images`.

Reply:
Done